conf.yaml

```diff
@@ -1031,7 +1031,7 @@ contents:
             prefix:     en/ecs
             # Please do not update current to 8.18
             current:    8.17
```

Comment:
@lcawl , @bmorelli25 : I have a question here... are we going to keep ECS current in 8.18 even after releasing 8.18? why do we have that comment to not update current to 8.18? And why we don't have current = stackcurrent?

Reply:
The ECS folks manage that book and just bumped it recently in https://github.com/elastic/docs/pull/3157 it doesn't follow the same cadence as the other stack-versioned docs.